internal/icinga2/client_api.go

```diff
@@ -130,6 +130,31 @@ func (client *Client) queryObjectsApiDirect(ctx context.Context, objType, objNam
 		map[string]string{"Accept": "application/json"})
 }
 
+// fetchIcingaAppStatus retrieves the global state of the IcingaApplication type via the /v1/status endpoint.
+func (client *Client) fetchIcingaAppStatus(ctx context.Context) (*IcingaAppStatus, error) {
```

Comment:
The position of this method does not fit right. It's currently located between between the two `queryObjectsApiDirect` and `queryObjectsApiQuery` methods. Please move it after the `queryObjects..` methods to the other `fetch..` methods.